### PR TITLE
feat(wallet): add responsive menu

### PR DESCRIPTION
**Pull request by raducristianpopa**
Adds responsive menu (added a simple icon to every item in the menu).

<details>
<summary>Desktop</summary>
image
</details>

<details>
<summary>Mobile</summary>
image
</details>

### CODE REVIEW

File: packages/wallet/src/components/Menu.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  },
+  {
+    name: 'Settings',
+    href: '/settings',
+    icon: <Home className="h-6 w-6" />
+  }
+]
+
+export const Menu = () => {
+  const { pathname } = useRouter()
```
Comment (Tymmmy) [inline packages/wallet/src/components/Menu.tsx:46]: pathname should be pathName

File: packages/wallet/src/components/Menu.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+  return (
+    <Disclosure as="nav">
+      {({ open }) => (
+        <>
+          <div className="fixed inset-x-0 flex h-20 flex-col md:inset-y-0 md:h-auto md:w-60">
```
Comment (Tymmmy) [inline packages/wallet/src/components/Menu.tsx:52]: the 2 divs act like containers for the menus, both are flex layouts, could they be merged?

File: packages/wallet/src/components/Menu.tsx (hunk 1 above)
Comment (raducristianpopa) [reply packages/wallet/src/components/Menu.tsx:46]: This is what `useRouter` is returning.

https://nextjs.org/docs/api-reference/next/router#router-object

File: packages/wallet/src/components/Menu.tsx (hunk 2 above)
Comment (raducristianpopa) [reply packages/wallet/src/components/Menu.tsx:52]: The first `div` acts as a wrapper, but we can use the `nav` element as the wrapper and remove the extra `div`.

**Review by Tymmmy — APPROVED** [review]

**Pull request merged**